### ISSUE TITLE
Cannot access /dev/hidraw error as a Likewise-open user

### ISSUE THREAD

**Issue by darnells** [pwr/Solaar#177]
When logged in as a Likewise-open (or PowerBroker Identity Services open) user, I receive an error message that Solaar cannot access the device /dev/hidraw3. These users are not conventional local users; they are authenticated against a Windows domain server. I added my Likewise user to the local plugindev group, but the error message persisted. Solaar worked when I logged in as a conventional local user.

I worked around the issue using a suggestion at http://unix.stackexchange.com/a/85459/39110. From the thread:

The following line was placed in a .rules file (I named mine 99-hidraw-permissions.rules) located under /etc/udev/rules.d.

KERNEL=="hidraw*", SUBSYSTEM=="hidraw", MODE="0664", GROUP="plugdev"

Basically this assigns all devices coming out of the hidraw subsystem in the kernel to the group plugdev and sets the permissions to r/w r/w r (for root [the default owner], plugdev, and everyone else respectively).

Could the default Solaar udev rule be modified in a similar way to ensure all plugdev group members have access to the Logitech Unifying receiver?

**Comment by jrbenito** [comment]
@darnells and @Lekensteyn 

Long time has passed, is this bug still valid?

**Comment by Lekensteyn** [comment]
It is a very specific problem which are probably not encountered by the majority of users.

I would expect that the "uaccess" udev tag should work. If not, that would break modern desktops since they rely on this. I'll close this for now, please re-open if you are still having trouble that can be fixed in Solaar.
